handler/interceptors.go

```diff
@@ -130,6 +130,10 @@ type PaymentHandler interface {
 	Complete(payment Payment) (err *GrpcError)
 	// CompleteAfterError completes payment if service returns error.
 	CompleteAfterError(payment Payment, result error) (err *GrpcError)
+
+	//Publishing latest Authorized amount on a given channel ( to show the effective balance on DAPP/platform supported clients)
+	//Force this behaviour for any new handlers on posting the offline channel usage stats
+	PublishChannelStats(payment Payment) (err *GrpcError)
```

Comment:
Separate method in the interface is not needed. The idea of the interface is to do this work in `Complete` and `CompleteAfterError` methods. I would add `PublishChannelStats` to `paymentChannelPaymentHandler` only and call it from `Complete` and `CompleteAfterError`.

Reply:
Thanks @vsbogd , changed this , however dont see a need to call this from CompleteAfterError as the ETCD would not have been changed no ?

Reply:
Hi Anand, thanks, will look at this. 

> however dont see a need to call this from CompleteAfterError as the ETCD would not have been changed no ?

May be, I didn't look at the PR accurately just made the most major comment I had. 

Regarding your previous question how to enforce behavior: my opinion it is not required to enforce it for absolutely all handlers. For example free-call handler doesn't need implementing it. If there is some subclass of handlers to enforce this we can implement sub-interface which inherits `PaymentHandler` (for example ChannelStatsPublishingPaymentHandler) and add this method into it. But I would start from implementing this logic in `Complete` and `CompleteAfterError`.